apps/web/tsconfig.json

```
@@ -1,6 +1,31 @@
 {
   "extends": "@calcom/tsconfig/nextjs.json",
   "compilerOptions": {
+    "jsx": "react",
```

Biome (1.9.4)

[error] The key jsx was already declared.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)